tests/core/policies/test_ted_policy.py

```diff
@@ -12,8 +12,12 @@
     MaxHistoryTrackerFeaturizer,
```

Comment:
💡 Please don't be intimidated by the amount of changes in the test file. This was only some extra work we had to do for the first policy due to our `PolicyTestCollection`. For your policy you will only have to migrate the `test_xy_policy` part.